xml/tuning_tuned.xml

```diff
@@ -0,0 +1,2501 @@
+        <para>
+          The <package>tuned</package> package is available only in the official
+          <emphasis>&sles;</emphasis> repositories. There is no equivalent package available in the
+          official &sled; repositories.
+        </para>
+      </note>
+      <procedure>
+        <step>
+          <para>
+            To install the &tuned; daemon, command-line utilities, profiles and plug-ins, run the
+            following command:
+          </para>
+<screen>&prompt.sudo;<command>zypper install -y tuned</command></screen>
+        </step>
+        <step>
+          <para>
+            Verify the installation by running the following commands:
+          </para>
+<screen>&prompt.user;<command>tuned --help</command></screen>
+<screen>&prompt.user;<command>tuned-adm --help</command></screen>
+        </step>
+        <step>
+          <para>
+            <emphasis>(OPTIONAL)</emphasis> To know more, read the <emphasis>man pages</emphasis>
+            for <literal>tuned</literal>, <literal>tuned-adm</literal>,
+            <literal>tuned-profiles</literal> and <literal>tuned.conf</literal>.
+          </para>
+        </step>
+      </procedure>
+    </sect2>
+
+    <sect2 xml:id="sec-tuning-tuned-enable">
+      <title>Enabling &tuned;</title>
+      <para>
+        To enable &tuned;, perform the following procedure:
+      </para>
+      <procedure>
+        <step>
+          <para>
+            Check the status of the &tuned; service:
+          </para>
+<screen>&prompt.sudo;<command>systemctl status tuned</command></screen>
+          <para>
+            By default, the status of the service is <emphasis>disabled</emphasis>.
+          </para>
+        </step>
+        <step>
+          <para>
+            To enable a systemd service for future boots so that the &tuned; daemon starts
+            automatically at the boot time, run the following command:
+          </para>
+<screen>&prompt.sudo;<command>systemctl enable tuned</command></screen>
+          <para>
+            However, this command does not starts the &tuned; process. If you check the status of
+            the systemd service now, it should be <emphasis>enabled</emphasis> and
+            <emphasis>inactive</emphasis>.
+          </para>
+        </step>
+      </procedure>
+    </sect2>
+
+    <sect2 xml:id="sec-tuning-tuned-start">
+      <title>Starting &tuned;</title>
+      <para>
+        After enabling the associated systemd service for &tuned;, for the current session you can
+        start &tuned; either as a background daemon or as a normal process connected to the TTY.
+      </para>
+      <para>
+        Alternatively, you can enable the associated systemd service and start the daemon
+        simultaneously.
+      </para>
+      <procedure>
+        <step>
+          <para>
+            To start &tuned; only for the current session, perform one of the following steps:
+          </para>
+          <itemizedlist>
+            <listitem>
```

Comment:
If you are already in a procedure and have additional steps inside another step, use `<substeps>`:

```suggestion
          <substeps>
            <step>
```

Add the necessary end tags. :wink: